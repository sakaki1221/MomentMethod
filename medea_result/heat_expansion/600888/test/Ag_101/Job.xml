<?xml version='1.0'?>
<!DOCTYPE MDRESULTS PUBLIC "-//MD//DTD MDRESULTS 1.0" "http://materialsdesign.com/DTD/mdresults_1_0.dtd" []>

<MDRESULTS>
  <System id="initial">
    <Date>16 Jan 2017 05:55:17 GMT</Date>
  <Characterization><PropertyValue property="symmetry" type="text" units="">P1</PropertyValue><PropertyValue property="setting" type="text" units="">P1</PropertyValue><PropertyValue property="empirical formula" type="text" units="">Ag</PropertyValue><PropertyValue property="primitive cell formula" type="text" units="">Ag32</PropertyValue><PropertyValue property="primitive Z" type="text" units="">32</PropertyValue><PropertyValue property="symmetry" type="text" units="">P1</PropertyValue><PropertyValue property="setting" type="text" units="">P1</PropertyValue><PropertyValue property="empirical formula" type="text" units="">Ag</PropertyValue><PropertyValue property="primitive cell formula" type="text" units="">Ag32</PropertyValue><PropertyValue property="primitive Z" type="text" units="">32</PropertyValue></Characterization></System>
  <Calculation id="initial">
    <Date>16 Jan 2017 05:55:17 GMT</Date>
    <System>initial</System>
  <Results><PropertyValue property="VASP energy" type="float" units="eV">-90.333933</PropertyValue><PropertyValue property="Fermi energy" type="float" units="eV">2.829300</PropertyValue><PropertyValue dimensions="3 32" property="atomic derivatives" type="float" units="kJ/mol/m">0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000</PropertyValue><PropertyValue property="cell volume" type="float" units="Ang^3">595.000</PropertyValue><PropertyValue property="pressure" type="float" units="GPa">-2.508</PropertyValue><PropertyValue dimensions="6" property="stress" type="float" units="GPa">2.50800 2.50800 2.50800 0.00000 0.00000 0.00000</PropertyValue><PropertyValue property="energy of formation" type="float" units="kJ/mol">0.4543</PropertyValue><PropertyValue property="heat of formation" type="float" units="kJ/mol">-27.6288</PropertyValue><PropertyValue property="VASP energy" type="float" units="eV">-90.333933</PropertyValue><PropertyValue property="Fermi energy" type="float" units="eV">2.829300</PropertyValue><PropertyValue dimensions="3 32" property="atomic derivatives" type="float" units="kJ/mol/m">0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000</PropertyValue><PropertyValue property="cell volume" type="float" units="Ang^3">595.000</PropertyValue><PropertyValue property="pressure" type="float" units="GPa">-2.508</PropertyValue><PropertyValue dimensions="6" property="stress" type="float" units="GPa">2.50800 2.50800 2.50800 0.00000 0.00000 0.00000</PropertyValue><PropertyValue property="energy of formation" type="float" units="kJ/mol">0.4543</PropertyValue><PropertyValue property="heat of formation" type="float" units="kJ/mol">-27.6288</PropertyValue></Results></Calculation>
<System id="displaced_Ag__a"><Date>18 Jan 2017 04:54:45 GMT</Date><Characterization><PropertyValue property="symmetry" type="text" units="">P1</PropertyValue><PropertyValue property="setting" type="text" units="">P1</PropertyValue><PropertyValue property="empirical formula" type="text" units="">Ag</PropertyValue><PropertyValue property="primitive cell formula" type="text" units="">Ag32</PropertyValue><PropertyValue property="primitive Z" type="text" units="">32</PropertyValue></Characterization></System><Calculation id="displaced_Ag__a">
    <Date>18 Jan 2017 04:54:45 GMT</Date>
    <System>displaced_Ag__a</System>
    <Results>
      <PropertyValue property="VASP energy" type="float" units="eV">-90.333177</PropertyValue>
      <PropertyValue property="Fermi energy" type="float" units="eV">2.829200</PropertyValue>
      <PropertyValue dimensions="3 32" property="atomic derivatives" type="float" units="kJ/mol/m">50819092.565 13995325.788 0.000
1051026001.092 1224132143.328 0.000
1051026045.983 22.445 -1224132143.328
50819093.078 -13995325.788 0.000
-209012160.542 3785620.979 -3785620.910
-78236165.472 0.000 0.000
51048524.644 -13765894.218 0.000
1051026001.092 -22.445 1224132143.328
1051026045.983 -1224132143.328 0.000
50819093.078 0.257 -13995325.788
-209012160.681 3785620.841 3785620.910
-8676987272.906 0.000 0.000
-209012160.403 -3785620.841 -3785620.910
-21681283.393 0.000 0.000
1125706023.970 -1325311465.830 0.000
1125705975.369 -24.300 1325311465.830
51048524.139 13765894.218 0.000
136855931.683 -688294.724 688294.711
-98655575.228 0.000 0.000
50819092.565 -0.257 13995325.788
-209012160.542 -3785620.979 3785620.910
-78236165.472 0.000 0.000
1125706023.970 24.300 -1325311465.830
1125705975.369 1325311465.830 0.000
51048524.139 -0.252 13765894.218
136855931.708 -688294.698 -688294.711
257078074.520 0.000 0.000
136855931.658 688294.698 688294.711
-30055535.709 0.000 0.000
51048524.644 0.252 -13765894.218
136855931.683 688294.724 -688294.711
-98655575.228 0.000 0.000</PropertyValue>
      <PropertyValue property="cell volume" type="float" units="Ang^3">595.000</PropertyValue>
      <PropertyValue property="pressure" type="float" units="GPa">-2.477</PropertyValue>
      <PropertyValue dimensions="6" property="stress" type="float" units="GPa">2.47600 2.47800 2.47800 0.00000 0.00000 0.00000</PropertyValue>
      <PropertyValue property="energy of formation" type="float" units="kJ/mol">0.4566</PropertyValue>
      <PropertyValue property="heat of formation" type="float" units="kJ/mol">-27.2794</PropertyValue>
    </Results>
  </Calculation><System id="displaced_Ag__b"><Date>18 Jan 2017 04:54:46 GMT</Date><Characterization><PropertyValue property="symmetry" type="text" units="">P1</PropertyValue><PropertyValue property="setting" type="text" units="">P1</PropertyValue><PropertyValue property="empirical formula" type="text" units="">Ag</PropertyValue><PropertyValue property="primitive cell formula" type="text" units="">Ag32</PropertyValue><PropertyValue property="primitive Z" type="text" units="">32</PropertyValue></Characterization></System><Calculation id="displaced_Ag__b">
    <Date>18 Jan 2017 04:54:46 GMT</Date>
    <System>displaced_Ag__b</System>
    <Results>
      <PropertyValue property="VASP energy" type="float" units="eV">-90.333177</PropertyValue>
      <PropertyValue property="Fermi energy" type="float" units="eV">2.829200</PropertyValue>
      <PropertyValue dimensions="3 32" property="atomic derivatives" type="float" units="kJ/mol/m">-51163239.926 -13651178.433 0.000
-1125705975.374 -1325082034.259 0.000
-1125706023.966 -24.296 1325082034.259
-51163240.427 13651178.433 0.000
209585739.468 3785620.979 -3785620.910
78350881.257 0.000 0.000
-50933808.863 13995325.788 0.000
-1125705975.374 24.296 -1325082034.259
-1125706023.966 1325082034.259 0.000
-51163240.427 -0.250 13651178.433
209585739.329 3785620.841 3785620.910
8674463525.632 0.000 0.000
209585739.607 -3785620.841 -3785620.910
21566567.608 0.000 0.000
-1050911330.193 1223902711.758 0.000
-1050911285.311 22.441 -1223902711.758
-50933808.350 -13995325.788 0.000
-136970647.468 -688294.724 688294.711
99458585.724 0.000 0.000
-51163239.926 0.250 -13651178.433
209585739.468 -3785620.979 3785620.910
78350881.257 0.000 0.000
-1050911330.193 -22.441 1223902711.758
-1050911285.311 -1223902711.758 0.000
-50933808.350 0.257 -13995325.788
-136970647.443 -688294.698 -688294.711
-257766369.230 0.000 0.000
-136970647.493 688294.698 688294.711
30170251.494 0.000 0.000
-50933808.863 -0.257 13995325.788
-136970647.468 688294.724 -688294.711
99458585.724 0.000 0.000</PropertyValue>
      <PropertyValue property="cell volume" type="float" units="Ang^3">595.000</PropertyValue>
      <PropertyValue property="pressure" type="float" units="GPa">-2.477</PropertyValue>
      <PropertyValue dimensions="6" property="stress" type="float" units="GPa">2.47600 2.47800 2.47800 0.00000 0.00000 0.00000</PropertyValue>
      <PropertyValue property="energy of formation" type="float" units="kJ/mol">0.4566</PropertyValue>
      <PropertyValue property="heat of formation" type="float" units="kJ/mol">-27.2794</PropertyValue>
      <PropertyValue property="zero-point energy" type="float" units="kJ/mol">1.6847</PropertyValue>
      <PropertyValue property="E(0)" type="float" units="kJ/mol">2.1390</PropertyValue>
      <PropertyValue property="E(PV)" type="float" units="kJ/mol">-28.0831</PropertyValue>
      <PropertyValue property="Eelec" type="float" units="kJ/mol">0.4543</PropertyValue>
      <PropertyValue property="Temperatures" type="float" units="K">0.001 1 2 3 4 5 10 15 20 30 40 50 60 70 80 90 100 125 150 175 200 225 250 273 275 298 300 350 400 450 500 550 600 650 700 750 800 850 900 950 1e+003 1.05e+003 1.1e+003 1.15e+003 1.2e+003 1.25e+003 1.3e+003 1.35e+003 1.4e+003 1.45e+003 1.5e+003 1.55e+003 1.6e+003 1.65e+003 1.7e+003 1.75e+003 1.8e+003 1.85e+003 1.9e+003 1.95e+003 2e+003 2.1e+003 2.2e+003 2.3e+003 2.4e+003 2.5e+003 2.6e+003 2.7e+003 2.8e+003 2.9e+003 3e+003</PropertyValue>
      <PropertyValue property="S(T)" type="float" units="J/K/mol">0.0000 0.0000 0.0005 0.0024 0.0062 0.0120 0.0829 0.3102 0.8136 2.6624 5.2188 8.0277 10.8348 13.5258 16.0586 18.4249 20.6309 25.5263 29.6975 33.3112 36.4899 39.3228 41.8752 44.0183 44.1965 46.1602 46.3240 50.1091 53.4008 56.3119 58.9207 61.2838 63.4434 65.4315 67.2733 68.9888 70.5942 72.1028 73.5254 74.8714 76.1487 77.3638 78.5225 79.6298 80.6901 81.7072 82.6846 83.6250 84.5314 85.4060 86.2509 87.0683 87.8597 88.6268 89.3709 90.0936 90.7959 91.4790 92.1439 92.7915 93.4228 94.6394 95.7993 96.9077 97.9690 98.9869 99.9650 100.9062 101.8131 102.6882 103.5337</PropertyValue>
      <PropertyValue property="Cv(T)" type="float" units="kJ/mol">0.0000 0.0000 0.0022 0.0086 0.0190 0.0338 0.2497 1.0576 2.6276 6.8826 10.9637 14.1798 16.5622 18.3068 19.5946 20.5599 21.2961 22.5080 23.2116 23.6525 23.9456 24.1498 24.2976 24.4001 24.4078 24.4862 24.4922 24.6107 24.6881 24.7414 24.7796 24.8079 24.8295 24.8463 24.8597 24.8705 24.8793 24.8866 24.8928 24.8980 24.9024 24.9062 24.9095 24.9124 24.9150 24.9172 24.9192 24.9210 24.9226 24.9240 24.9253 24.9264 24.9275 24.9284 24.9293 24.9301 24.9309 24.9315 24.9322 24.9327 24.9333 24.9342 24.9351 24.9358 24.9364 24.9370 24.9375 24.9379 24.9383 24.9387 24.9390</PropertyValue>
      <PropertyValue property="H(T)-Eelec" type="float" units="kJ/mol">-26.3984 -26.3984 -26.3984 -26.3984 -26.3984 -26.3984 -26.3978 -26.3949 -26.3859 -26.3389 -26.2491 -26.1226 -25.9683 -25.7935 -25.6036 -25.4026 -25.1932 -24.6442 -24.0720 -23.4858 -22.8906 -22.2892 -21.6836 -21.1235 -21.0747 -20.5124 -20.4634 -19.2356 -18.0030 -16.7672 -15.5291 -14.2894 -13.0485 -11.8066 -10.5639 -9.3206 -8.0769 -6.8328 -5.5883 -4.3435 -3.0985 -1.8533 -0.6079 0.6376 1.8833 3.1291 4.3750 5.6210 6.8671 8.1133 9.3595 10.6058 11.8521 13.0985 14.3449 15.5914 16.8379 18.0845 19.3311 20.5777 21.8243 24.3177 26.8111 29.3047 31.7983 34.2919 36.7856 39.2794 41.7732 44.2670 46.7609</PropertyValue>
      <PropertyValue property="E(T)-Eelec" type="float" units="">1.6847 1.6847 1.6847 1.6847 1.6847 1.6847 1.6853 1.6882 1.6972 1.7442 1.8340 1.9605 2.1148 2.2896 2.4795 2.6805 2.8899 3.4389 4.0111 4.5973 5.1925 5.7939 6.3996 6.9596 7.0084 7.5707 7.6197 8.8475 10.0801 11.3159 12.5540 13.7937 15.0346 16.2766 17.5192 18.7625 20.0062 21.2503 22.4948 23.7396 24.9846 26.2298 27.4752 28.7207 29.9664 31.2122 32.4581 33.7041 34.9502 36.1964 37.4426 38.6889 39.9352 41.1816 42.4280 43.6745 44.9210 46.1676 47.4142 48.6608 49.9074 52.4008 54.8942 57.3878 59.8814 62.3750 64.8687 67.3625 69.8563 72.3501 74.8440</PropertyValue>
    </Results>
  </Calculation></MDRESULTS>
