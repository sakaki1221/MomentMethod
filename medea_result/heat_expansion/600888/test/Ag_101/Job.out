Startup is 'sourced'
Opening the database
Phonon calculation for Untitled

Phonon parameters
=================
The calculation explores the full Brillouin zone and accounts for an interaction range of about 10.0 Angstroem.
The asymmetric atoms are displaced by +/-0.02 Angstroem.

In a first step, need to run an optimization of atom positions for the supercell containing 32 atoms.
Then, need to run a total number of 2 single point energy calculations on supercells with displaced atoms containing 32 atoms.

      Supercell transformation matrix              Supercell lattice vectors
       2.00000    0.00000    0.00000             8.41081    0.00000    0.00000
       0.00000    2.00000    0.00000             0.00000    8.41081    0.00000
       0.00000    0.00000    2.00000             0.00000    0.00000    8.41081

     Supercell           Site        Displaced atom     Displacement (fractional units)
   ---------------    ----------     --------------     -------------------------------
  undisplaced           ----            ----             0.00000    0.00000    0.00000
  displaced_Ag__a       Ag              Ag               0.00238    0.00000    0.00000
  displaced_Ag__b       Ag              Ag              -0.00238    0.00000    0.00000

Invariance conditions are not enforced.
No approximation to LO/TO splitting
Properties: Phonon dispersion and form factors 
            Phonon density of states 
            Thermodynamic functions
==========================================


VASP parameters
===============
Since no magnetic moments are in the model, this is a non-magnetic calculation using 'accurate' precision
and a user-defined planewave cutoff energy of 600.000 eV.

The electronic iterations convergence is 1.00E-005 eV using the Very Fast (RMM-DIIS) algorithm
and reciprocal space projection operators.

The requested k-spacing is 0.1 per Angstrom, which leads to a 8x8x8 mesh.
This corresponds to actual k-spacings of 0.093 x 0.093 x 0.093 per Angstrom.

Using first order Methfessel-Paxton smearing with a width of 0.2 eV.

Other non-default parameters:
	Extrafine augmentation grid for accurate forces is TRUE
	Algorithm is Very Fast (RMM-DIIS)
	Wave functions is TRUE
==========================================

------------------------------------------------------------------------

Results for initial structural minimization of the supercell without displacements:
--------------------------------------------------------------------------------------

There are 20 symmetry-unique k-points
The plane wave cutoff is 600.00 eV

   VASP energy:             -90.333933 eV for Ag32

   Initial VASP energy:     -90.333933 eV for Ag32
   Relaxation energy:         0.000000 eV gained after 1 optimization steps.

Electronic contributions:
                          Empirical Formula          Cell
                                    Ag          (Ag)32
                          --------------- ---------------
              VASP Energy        -272.372       -8715.897 kJ/mol
      Energy of formation           0.454          14.538
                  PV term         -28.083        -898.659

   This is the electronic part of the formation energy with respect to the
   elements in their standard state. It is obtained from the total energy (above)
   and tabulated VASP energies for the elements in their standard state.
   See the documentation for more details.

   Details for Ag:
                                             VASP energy (Etot):     -272.37 kJ/mol
       Standard state energy of 1.00 atoms of Ag  1.00 * -272.83:  -  -272.83
                                                                   ----------
                                        Energy of formation (Ef):        0.45 kJ/mol


Cell parameters:
 Parameter    Value    
---------- ------------
         a     8.410814
         b     8.410814
         c     8.410814
     alpha    89.999999
      beta    89.999999
     gamma    89.999999

    Volume    74.374497 Ang^3

        Density:     9.633 Mg/m^3

   The pressure given below is exerted by the system according to its volume.
   Positive pressure would cause expansion during full geometry optimization.
       Pressure:    -2.508 GPa
               =   -25.080 kbar

                      XX        YY        ZZ        XY        YZ        ZX
         Stress:     2.508     2.508     2.508     0.000     0.000     0.000 GPa
               =    25.080    25.080    25.080     0.000     0.000     0.000 kbar
   The stress tensor above is imposed on the system, i.e. negative values of diagonal components 
   would cause expansion of the corresponding lattice parameter upon full geometry optimization.
   The pressure and stress include only electronic terms, i.e. the vibrational,
   temperature and other terms are not included here.

Fractional Coordinates:
	Atom          Initial Coordinates             Final Coordinates
	-----      -------- -------- --------      -------- -------- --------
	Ag           0.7500   0.7500   0.5000        0.7500   0.7500   0.5000
	Ag           0.7500   0.7500   0.0000        0.7500   0.7500   0.0000
	Ag           0.7500   0.0000   0.2500        0.7500   0.0000   0.2500
	Ag           0.7500   0.2500   0.5000        0.7500   0.2500   0.5000
	Ag           0.0000   0.7500   0.2500        0.0000   0.7500   0.2500
	Ag           0.0000   0.0000   0.5000        0.0000   0.0000   0.5000
	Ag           0.2500   0.7500   0.5000        0.2500   0.7500   0.5000
	Ag           0.7500   0.0000   0.7500        0.7500   0.0000   0.7500
	Ag           0.7500   0.2500   0.0000        0.7500   0.2500   0.0000
	Ag           0.7500   0.5000   0.2500        0.7500   0.5000   0.2500
	Ag           0.0000   0.7500   0.7500        0.0000   0.7500   0.7500
	Ag           0.0000   0.0000   0.0000        0.0000   0.0000   0.0000
	Ag           0.0000   0.2500   0.2500        0.0000   0.2500   0.2500
	Ag           0.0000   0.5000   0.5000        0.0000   0.5000   0.5000
	Ag           0.2500   0.7500   0.0000        0.2500   0.7500   0.0000
	Ag           0.2500   0.0000   0.2500        0.2500   0.0000   0.2500
	Ag           0.2500   0.2500   0.5000        0.2500   0.2500   0.5000
	Ag           0.5000   0.7500   0.2500        0.5000   0.7500   0.2500
	Ag           0.5000   0.0000   0.5000        0.5000   0.0000   0.5000
	Ag           0.7500   0.5000   0.7500        0.7500   0.5000   0.7500
	Ag           0.0000   0.2500   0.7500        0.0000   0.2500   0.7500
	Ag           0.0000   0.5000   0.0000        0.0000   0.5000   0.0000
	Ag           0.2500   0.0000   0.7500        0.2500   0.0000   0.7500
	Ag           0.2500   0.2500   0.0000        0.2500   0.2500   0.0000
	Ag           0.2500   0.5000   0.2500        0.2500   0.5000   0.2500
	Ag           0.5000   0.7500   0.7500        0.5000   0.7500   0.7500
	Ag           0.5000   0.0000   0.0000        0.5000   0.0000   0.0000
	Ag           0.5000   0.2500   0.2500        0.5000   0.2500   0.2500
	Ag           0.5000   0.5000   0.5000        0.5000   0.5000   0.5000
	Ag           0.2500   0.5000   0.7500        0.2500   0.5000   0.7500
	Ag           0.5000   0.2500   0.7500        0.5000   0.2500   0.7500
	Ag           0.5000   0.5000   0.0000        0.5000   0.5000   0.0000

Results for supercell calculations:
------------------------------------------------------------------------
	        Supercell         Energy (eV)
	      -------------      -------------
	        undisplaced       -90.33393
	     displaced_Ag__a      -90.33318
	     displaced_Ag__b      -90.33318


Symmetry analysis of Gamma point phonon frequencies:
----------------------------------------------------
   Crystal  Point  Group: O_h   (m3m) 
   Supercell Point Group: O_h   (m3m) 
    Table of irreducible representations of point group:
 ------------------------------------------
    IR     1  5  7  9 11  6  8 10 12 13 16 17 18 21 23 14 15 19 20 22 24  2  3  4 25 38 39 43 44 46 48 29 30 31 32 33 34 35 36 26 27 28 37 40 41 42 45 47
 ------------------------------------------
 A1g(R)    1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1   x^2+y^2+z^2                         
 A2g       1  1  1  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1  1  1  1  1 -1 -1 -1 -1 -1 -1  1  1  1  1  1  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1                                       
 Eg(R)     2 -1 -1 -1 -1 -1 -1 -1 -1  0  0  0  0  0  0  0  0  0  0  0  0  2  2  2  2  0  0  0  0  0  0 -1 -1 -1 -1 -1 -1 -1 -1  2  2  2  0  0  0  0  0  0   (2z^2-x^2-y^2, x^2-y^2)             
 T1g       3  0  0  0  0  0  0  0  0 -1 -1 -1 -1 -1 -1  1  1  1  1  1  1 -1 -1 -1  3  1  1  1  1  1  1  0  0  0  0  0  0  0  0 -1 -1 -1 -1 -1 -1 -1 -1 -1                                       
 T2g(R)    3  0  0  0  0  0  0  0  0  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1 -1 -1 -1  3 -1 -1 -1 -1 -1 -1  0  0  0  0  0  0  0  0 -1 -1 -1  1  1  1  1  1  1   (xy,xz,yz)                          
 A1u       1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1                                       
 A2u       1  1  1  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1  1  1  1 -1  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1  1  1  1  1  1  1                                       
 Eu        2 -1 -1 -1 -1 -1 -1 -1 -1  0  0  0  0  0  0  0  0  0  0  0  0  2  2  2 -2  0  0  0  0  0  0  1  1  1  1  1  1  1  1 -2 -2 -2  0  0  0  0  0  0                                       
 T1u(I)    3  0  0  0  0  0  0  0  0 -1 -1 -1 -1 -1 -1  1  1  1  1  1  1 -1 -1 -1 -3 -1 -1 -1 -1 -1 -1  0  0  0  0  0  0  0  0  1  1  1  1  1  1  1  1  1   (x,y,z)                             
 T2u       3  0  0  0  0  0  0  0  0  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -3  1  1  1  1  1  1  0  0  0  0  0  0  0  0  1  1  1 -1 -1 -1 -1 -1 -1                                       
 ------------------------------------------
    Wave vector= 0.0000  0.0000  0.0000    no macro. electric field
 Multi. Omega   Irreducible Representations
  3    -0.018     T1u(I)                                                                                                     
Notation: (R)  Raman active mode
          (I)  Infrared active mode
          (RI) Raman and Infrared active mode

Thermodynamic Properties
   for empirical formula Ag

The following thermodynamic functions are obtained from the phonon
density of states within the harmonic approximation. The results are
meaningful only below the melting point, which lies well below 3000 K
for most materials.
Definitions:
      Cv            : vibrational heat capacity at constant volume
E_vib(T)=E(T)-E(0)  : the change in vibrational internal energy from 0 K, where
                        E(0) = E_elec + ZPE is the electronic energy of formation E_elec plus the zero point energy ZPE
      S(T)          : the vibrational entropy at temperature T
   -(A(T)-E(0))     : the change in the vibrational Helmholtz free energy from 0 K
      E(T)          : the electronic energy of formation plus the vibrational internal energy, so this term is
                        E_elec + ZPE + E_vib(T)
      A(T)          : the electronic plus vibrational Helmholtz free energy,
                        E(T) - T.S(T)
Note that the electronic energy, which is part of the enthalpy and free energies,
 is referenced to the elements in their standard state.

Electronic energy =     0.45 kJ/mol (referenced to elements in the standard state)
          PV term =   -28.08 kJ/mol
Zero-point energy =     1.68 kJ/mol

    T       Cv       E(T)-E(0)      S(T)   -(A(T)-E(0))     E(T)        A(T)
    K     J/K/mol      kJ/mol     J/K/mol     kJ/mol       kJ/mol      kJ/mol
  ----- ----------- ----------- ----------- ----------- ----------- -----------
     0      0.0000      0.0000      0.0000      0.0000      2.1390      2.1390
     1      0.0000      0.0000      0.0000      0.0000      2.1390      2.1390
     2      0.0022      0.0000      0.0005      0.0000      2.1390      2.1390
     3      0.0086      0.0000      0.0024      0.0000      2.1390      2.1390
     4      0.0190      0.0000      0.0062      0.0000      2.1390      2.1390
     5      0.0338      0.0000      0.0120      0.0000      2.1390      2.1390
    10      0.2497      0.0006      0.0829      0.0002      2.1396      2.1388
    15      1.0576      0.0036      0.3102      0.0011      2.1425      2.1379
    20      2.6276      0.0125      0.8136      0.0038      2.1515      2.1352
    30      6.8826      0.0595      2.6624      0.0203      2.1985      2.1186
    40     10.9637      0.1494      5.2188      0.0594      2.2884      2.0796
    50     14.1798      0.2758      8.0277      0.1255      2.4148      2.0134
    60     16.5622      0.4302     10.8348      0.2199      2.5692      1.9191
    70     18.3068      0.6050     13.5258      0.3418      2.7439      1.7972
    80     19.5946      0.7948     16.0586      0.4899      2.9338      1.6491
    90     20.5599      0.9958     18.4249      0.6624      3.1348      1.4765
   100     21.2961      1.2052     20.6309      0.8579      3.3442      1.2811
   125     22.5080      1.7542     25.5263      1.4365      3.8932      0.7025
   150     23.2116      2.3265     29.6975      2.1281      4.4654      0.0108
   175     23.6525      2.9127     33.3112      2.9168      5.0516     -0.7778
   200     23.9456      3.5079     36.4899      3.7901      5.6468     -1.6511
   225     24.1498      4.1092     39.3228      4.7384      6.2482     -2.5994
   250     24.2976      4.7149     41.8752      5.7539      6.8539     -3.6149
   273     24.4001      5.2749     44.0183      6.7420      7.4139     -4.6030
   275     24.4078      5.3238     44.1965      6.8302      7.4627     -4.6912
   298     24.4862      5.8861     46.1602      7.8696      8.0251     -5.7306
   300     24.4922      5.9350     46.3240      7.9621      8.0740     -5.8231
   350     24.6107      7.1628     50.1091     10.3752      9.3018     -8.2362
   400     24.6881      8.3954     53.4008     12.9648     10.5344    -10.8258
   450     24.7414      9.6312     56.3119     15.7089     11.7702    -13.5700
   500     24.7796     10.8693     58.9207     18.5909     13.0083    -16.4519
   550     24.8079     12.1090     61.2838     21.5969     14.2480    -19.4579
   600     24.8295     13.3500     63.4434     24.7158     15.4890    -22.5768
   650     24.8463     14.5919     65.4315     27.9383     16.7309    -25.7994
   700     24.8597     15.8345     67.2733     31.2565     17.9735    -29.1175
   750     24.8705     17.0778     68.9888     34.6635     19.2168    -32.5245
   800     24.8793     18.3215     70.5942     38.1535     20.4605    -36.0145
   850     24.8866     19.5657     72.1028     41.7213     21.7047    -39.5823
   900     24.8928     20.8102     73.5254     45.3623     22.9491    -43.2233
   950     24.8980     22.0549     74.8714     49.0725     24.1939    -46.9335
  1000     24.9024     23.2999     76.1487     52.8483     25.4389    -50.7093
  1050     24.9062     24.5451     77.3638     56.6863     26.6841    -54.5473
  1100     24.9095     25.7905     78.5225     60.5836     27.9295    -58.4447
  1150     24.9124     27.0361     79.6298     64.5376     29.1751    -62.3987
  1200     24.9150     28.2817     80.6901     68.5458     30.4207    -66.4068
  1250     24.9172     29.5275     81.7072     72.6059     31.6665    -70.4669
  1300     24.9192     30.7735     82.6846     76.7158     32.9124    -74.5768
  1350     24.9210     32.0194     83.6250     80.8737     34.1584    -78.7347
  1400     24.9226     33.2655     84.5314     85.0777     35.4045    -82.9387
  1450     24.9240     34.5117     85.4060     89.3262     36.6507    -87.1872
  1500     24.9253     35.7579     86.2509     93.6178     37.8969    -91.4788
  1550     24.9264     37.0042     87.0683     97.9508     39.1432    -95.8118
  1600     24.9275     38.2505     87.8597    102.3241     40.3895   -100.1851
  1650     24.9284     39.4969     88.6268    106.7363     41.6359   -104.5974
  1700     24.9293     40.7434     89.3709    111.1863     42.8824   -109.0474
  1750     24.9301     41.9898     90.0936    115.6730     44.1288   -113.5340
  1800     24.9309     43.2364     90.7959    120.1953     45.3754   -118.0563
  1850     24.9315     44.4829     91.4790    124.7522     46.6219   -122.6132
  1900     24.9322     45.7295     92.1439    129.3428     47.8685   -127.2039
  1950     24.9327     46.9761     92.7915    133.9662     49.1151   -131.8272
  2000     24.9333     48.2228     93.4228    138.6216     50.3617   -136.4827
  2100     24.9342     50.7161     94.6394    148.0252     52.8551   -145.8862
  2200     24.9351     53.2096     95.7993    157.5475     55.3485   -155.4085
  2300     24.9358     55.7031     96.9077    167.1832     57.8421   -165.0442
  2400     24.9364     58.1967     97.9690    176.9274     60.3357   -174.7884
  2500     24.9370     60.6903     98.9869    186.7754     62.8293   -184.6365
  2600     24.9375     63.1840     99.9650    196.7233     65.3230   -194.5843
  2700     24.9379     65.6778    100.9062    206.7671     67.8168   -204.6281
  2800     24.9383     68.1716    101.8131    216.9033     70.3106   -214.7643
  2900     24.9387     70.6654    102.6882    227.1286     72.8044   -224.9896
  3000     24.9390     73.1593    103.5337    237.4398     75.2983   -235.3008

Job completed on Wed 18 January 2017 at 13:54:47 æ±äº¬ (æ¨™æº–æ™‚) after 5696 s (1:34:56)
